sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ActionDescriptorFactory.scala

```diff
@@ -152,11 +152,12 @@ private[impl] object ActionDescriptorFactory extends ComponentDescriptorFactory
       nameGenerator,
       messageCodec,
       serviceName,
+      serviceOptions = AclDescriptorFactory.serviceLevelAclAnnotation(component),
       component.getPackageName,
-      filterAndAddKalixOptions(springAnnotatedMethods, publicationTopicMethods)
-      ++ subscriptionValueEntityMethods
-      ++ combineByES(component.getName, subscriptionEventSourcedEntityMethods, messageCodec)
-      ++ combineByTopic(subscriptionTopicMethods)
-      ++ removeDuplicates(springAnnotatedMethods, publicationTopicMethods))
+      filterAndAddKalixOptions(springAnnotatedMethods, publicationTopicMethods) ++
+      subscriptionValueEntityMethods ++
+      combineByES(component.getName, subscriptionEventSourcedEntityMethods, messageCodec) ++
+      combineByTopic(subscriptionTopicMethods) ++
+      removeDuplicates(springAnnotatedMethods, publicationTopicMethods))
```

Comment:
That was not intentional.

The reason is that I moved this block to outside the params block because initially I was planning to run a `map` and I didn't want to map inside the param block.

If you try to assign this Seq concatenation to a val, you need to add the `++` at the end. So I did that. Later, I changed my mind and brought the seq concatenation back inside and forgot to revert the position of the `++`.  

I do prefer this variant though, but just a question of taste.

I will revert it because that will conflict with Cisco's PR.